### PR TITLE
Saucelabs

### PR DESCRIPTION
**Pull request by grahamscott**
@spmason I've had a go at wiring the project up to Saucelabs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 language: node_js
 
+env:
+  global:
+    - secure: IIEC4WqIlugoTq2N/nrZN9j5v0PU13Mo0XacDOKdr18b3cewDQ1n3eicLjVxU+FZh6U1gb1awdIOJ4bnNCgaeiMs2KjAvI8xxsGFwKn996vT2qmJRlOMyowic4Ru0QmkZ+eiHE1FTSfbG7np0WmRpVX788mgpaGJvj7OrQgzVxM=
```
Comment (codazzo) [inline .travis.yml]: What are these?

**Comment by codazzo** [comment]
Just that comment from me @grahamscott

File: .travis.yml (hunk 1 above)
Comment (grahamscott) [inline .travis.yml]: They're encrypted credentials for Travis to talk to Saucelabs:

https://docs.saucelabs.com/ci-integrations/travis-ci/

**Comment by codazzo** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (spmason) [inline .travis.yml]: Is it cool that they're on a public github repo?

Comment (codazzo) [inline .travis.yml]: Yeah looks like it https://docs.saucelabs.com/ci-integrations/travis-ci/#adding-credentials-for-a-public-github-repo

Comment (spmason) [inline .travis.yml]: :thumbsup:

**Pull request merged**